coderd/provisionerdserver/provisionerdserver_test.go

```diff
@@ -1804,6 +1804,168 @@ func TestNotifications(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("TemplateManualBuildFailed", func(t *testing.T) {
+		t.Parallel()
+
+		t.Run("NotifyTemplateOwner", func(t *testing.T) {
```

Comment:
Please add given/when/then

Reply:
In general, please try make your tests read sensibly and clearly.
Try highlight the part of the test which is significant; you've got a _lot_ of setup here, and you could make it clearer what is expected and why.

Tests are not just validation, they're _communication_. They tell a story about how you expect your code to run.
If it's just large chunks of text then it makes comprehension more difficult, and the intent of the test much harder to discern.